docs/api-endpoints.md

```diff
@@ -32,6 +32,7 @@
 - `GET /api/boards`
 - `POST /api/boards`
 - `GET /api/boards/:id`
+	Ken: What information does the above route give that the below route does not?
```

Comment:
Good question, the below route is unnecessary. The boards' pins will be rendered upon entering that route. I'll delete that route.